src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "PRETTY_NAME" | sed 's/PRETTY_NAME=//g' | sed 's/["]//g' | awk '{print $1}'`
```

Comment:
@rahulkotecha Can you please address the PR comments on networking scripts. Thanks!